lessons/basics/composition.md

```diff
@@ -21,11 +21,11 @@ Modules are the best way to organize functions into a namespace.  In addition to
 Let's look at a basic example:
 
 ``` elixir
-defmodule Example do
-  def greeting(name) do
-    ~s(Hello #{name}.)
-  end
-end
+iex> defmodule Example do
```

Comment:
Mornin' @kianmeng!  Delete the commit and force push to your branch, that should update this PR.  Let me know if you need a hand :+1: